requirements/base.txt

```diff
@@ -34,4 +34,3 @@ sorl-thumbnail==v12.4a1
 sure==1.2.9
 boto==2.38.0
 git+https://github.com/DemocracyClub/django-uk-political-parties.git#egg=django-uk-political-parties
-django-extensions
```

Comment:
I think this isn't actually removing django-extensions (and the commit it's reverting wasn't adding it!) - django-extensions is mentioned twice in this `requirements/base.txt` file, once here (without being pinned to a particular version) and once above where it's pinned to 1.4.8, so after this commit it's still required at that old version.

The current released version of pip (9.0.1) errors on trying to install from `requirements.txt` if there's the duplicate requirement present:

```
$ pip --version
pip 9.0.1 from /home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages (python 2.7)
$ pip install -r requirements.txt 
Double requirement given: django-extensions (from -r requirements/base.txt (line 37)) (already in django-extensions==1.4.8 (from -r requirements/base.txt (line 9)), name='django-extensions')
```

... which suggests that you're using an older version of pip (n.b. it's probably best to use the most recent version).  I'm not sure what the behaviour of an older version of pip would be with a duplicate requirement, but if the first or more specific metion is being used that _might_ account for the problem you're seeing. From trying just now, migrating from a newly created database works fine for me with `django-extensions==1.7.8`, but fails with `django-extensions==1.4.8` because that older version depends on South, which is deprecated from Django >= 1.7 and not in the requirements currently.

```
$ ./manage.py migrate
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 89, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 20, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 49, in __init__
    self.build_graph()
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 170, in build_graph
    self.load_disk()
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 105, in load_disk
    migration_module = import_module("%s.%s" % (module_name, migration_name))
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/mark/.virtualenvs/electionleaflets/local/lib/python2.7/site-packages/django_extensions/migrations/0001_empty.py", line 2, in <module>
    from south.v2 import SchemaMigration
ImportError: No module named south.v2
```
Was that the error you were seeing? (BTW, if it was this South-related error then this still doesn't quite make sense, because the patch in this PR would mean that you definitely get `django-extensions==1.4.8`, with which migrating will faill.)

It's useful having django-extensions installed (for `./manage.py shell_plus` in particular) so assuming what I've said makes sense given what you've seen, I think it'd be better to remove the unpinned version here, as in this pull request, but also change the earlier requirement to 1.7.8.